features/buyers/service_contracts/bulk_operations.feature

```diff
@@ -141,7 +142,8 @@ Feature: Service subscriptions bulk operations
 
   Scenario: Changing service plan throws an error
     Given the subscription will return an error when changing its plan
-    When item "Alice" is selected
+    When table is sorted by "Plan"
```

Comment:
This failure in #2428 was not a test order issue but in fact overall flakiness due to table sort order or lack thereof. I fixed anyway while on it.

Reply:
Hm, interesting, because ordering by Plan does not guarantee a specific order either, I'd say... (in case there are multiple entries with the same plan)

Reply:
The point is to get the 2 `Alice` entries ordered deterministically. So this step serves the purpose. These tests are not sensitive to the order of the rest of the entries.